helm/eth2signer/templates/service.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.service.enabled }}
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Release.Name }}
+  labels:
+    app: {{ template "eth2signer.name" . }}
+    chart: {{ template "eth2signer.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  type: ClusterIP
```

Comment:
Thinking this should take the value from Values so that the service type can be overridden i.e. change this to {{ .Values.service.port }}

Reply:
nice catch, updated